src/libraries/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.Bignum.cs

```diff
@@ -55,7 +57,8 @@ internal static byte[] ExtractBignum(IntPtr bignum, int targetSize)
             }
         }
 
-        private static unsafe byte[] ExtractBignum(SafeBignumHandle bignum, int targetSize)
+        [return: NotNullIfNotNull("bignum")]
```

Comment:
This is not valid.  The method can still return null if `bignum` is not null... if `bignum.IsInvalid` is true.

Reply:
Right, thanks